source/lmp/tests/test_lammps_compact.py

```
@@ -0,0 +1,388 @@
+            f"pair_style deepmd {model.resolve()} center_group qm "
+            "environment_cutoff 1.5 include_molecule yes",
+            "pair_coeff * *",
+            "run 0",
+            "variable compact_energy equal pe",
+            f'print "${{compact_energy}}" file {energy_file} screen no',
+            f"write_dump all custom {force_file} id fx fy fz modify sort id",
+        ]
+    )
+    input_file.write_text("\n".join(commands) + "\n")
+
+    result = sp.run(
```

Ruff (0.16.0)

[error] `subprocess` call: check for execution of untrusted input

(S603)